vault/changelog.release.2021-11-16.md

```diff
@@ -0,0 +1,55 @@
+---
+id: NWuXh9ZreX3XKI6NhRIBJ
+title: "0.69"
+desc: ''
+updated: 1637082185375
+created: 1637081497904
+---
+
+Dendron 0.69 has sprouted  🌱
+
+Updates include enhancements to using Lookup, troubleshooting schema errors, and tweaks to [[Native Workspaces|dendron.topic.workspace.native]].
+
+If you haven't taken a look at what Native Workspaces entail, they are a way to have Dendron docs within an existing project (such as a docs directory of a Python project).
+
+### Highlights
+- feat(workspace): native workspace enhancements ([[docs|dendron.topic.workspace.native]])
+  - Dendron will now search for `dendron.yml` in native workspaces even if `dendron.yml` is not in the root of the workspace.
+- enhance(lookup): lookups with a dot at the end now look up hierarchies without having exact postfix
+  - Having a dot at the end of a lookup will now look up child and descendant notes (bubbling up the child notes to the top) that match the query, but without requiring the full match of the prefix note.
+- fix(markdown): footnote definitions including links are rendered incorrectly
+
+### Everything Else
+- enhance(pods): add `configPath` and `query` parameters to pods ([[docs|dendron.topic.pod.cli]])
+- enhance(views): support setting `sort_order` in tree view ([[docs|dendron.topic.publish-legacy.configuration#sort_order]])
+- enhance(pods): include name of note with error when airtable pod encounters error
+- enhance(schema): Add `Go to File` navigation prompt when schema malformation errors appear
+- fix(workspace): apply enableUser/HashTags to broken wikilinks code action
+- fix(views): `nav_order` property not respected in tree view
+- fix(workspace): `Dendron: Vault add` avoids adding duplicate line
+
+#### Breaking Changes
+Note graph commands lose their `V2` suffix and are now simply `Dendron: Show Note Graph` and `Dendron: Show Schema Graph`. This only impacts functionality with the command palette.
+
+### Community
+
+#### General Updates
+
+- Refactoring of the Dendron wiki hierarchies took place last week. Links should still work, due to Dendron magic.
```

Comment:
would have this as an entire paragraph. maybe put into a scratch note and reference it. you can copy our discord announcement

---

The changes are described in this github discussion (https://github.com/dendronhq/dendron/discussions/1665) and represent a major rehaul of how our website is organized. 

I'm happy to announce that those changes are now live and you can see them by visiting https://wiki.dendron.so/ 

In terms of changes, running git stat yields the following after the refactor:

- dendron-site
    
    227 files changed, 1131 insertions(+), 950 deletions(-)
    

- org-private (internal vault used by the dendron team)
    
    132 files changed, 541 insertions(+), 198 deletions(-)
    

In total, Dendron refactored over 300 files and made over 2k line changes as part of this reorg. 
In any other tool, something like this might have taken weeks or months (or in all likelihood, not happened at all) and left-behind lots of 404s.
In Dendron, it took a few hours (most of the time was spent making additional content changes), and no active links were broken in this process (we did prune some unused pages).

Our goal with Dendron is to build a tool to help humans organize, reference, and work with any amount of knowledge. 
Today's exercise was a small demonstration of that. 

Thanks, everyone for your continued support and please leave feedback about the new organization in the github discussion or in the feedback channel 🙏

Reply:
Decided to add it under it's own header